oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -754,6 +820,9 @@ void pldm::responder::oem_ibm_platform::Handler::buildOEMPDR(
                                 PLDM_OEM_IBM_BOOT_SIDE_RENAME, repo);
     buildAllNumericEffecterPDR(this, PLDM_ENTITY_PROC, ENTITY_INSTANCE_0,
                                PLDM_OEM_IBM_SBE_SEMANTIC_ID, repo, instanceMap);
+    buildAllSystemPowerStateEffecterPDR(
+        this, PLDM_OEM_IBM_CHASSIS_POWER_CONTROLLER, ENTITY_INSTANCE_0,
+        PLDM_STATE_SET_SYSTEM_POWER_STATE, repo);
```

Comment:
This OEM entity should be present in the entity association PDR, please add it. Otherwise phyp cannot access this effecter.

Reply:
Done